docs/edge-device-doc.md

```diff
@@ -0,0 +1,95 @@
+#             DIDComm Messaging for Edge Device Registration and Message Exchange
+## Introduction
+With the growth of IoT and connected devices, ensuring secure, private communication between edge devices—like smart sensors, appliances, and mobile devices—has become essential. DIDComm (Decentralized Identifier Communication) provides a protocol for private, out-of-band (OOB) messaging, enabling communication without intermediaries directly accessing data. This makes it a powerful tool for IoT networks where privacy and security are paramount.
```

Comment:
**DIDComm provides a protocol for private, out-of-band (OOB) messaging** does didcomm really provides a and out of band messaging way ? It's important to initiate a didcomm communication but didcomm does not provides a way for this to be done